[PATCH] Pass various flags to GCC

* build-aux/ltmain.in (func_mode_link): Pass various flags to GCC.

See https://gcc.gnu.org/onlinedocs/gcc/Link-Options.html#DOCF1
---
 build-aux/ltmain.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/build-aux/ltmain.in b/build-aux/ltmain.in
index e3c77f29..0560159b 100644
--- a/build-aux/ltmain.in
+++ b/build-aux/ltmain.in
@@ -5460,6 +5460,7 @@ func_mode_link ()
       # -static-libsan       Link with static sanitizer runtimes (Clang)
       # -no-canonical-prefixes Do not expand any symbolic links
       # -fuse-ld=*           Linker select flags for GCC
+      # -shared-*            direct GCC to link specific libraries dynamically
       # -static-*            direct GCC to link specific libraries statically
       # -fcilkplus           Cilk Plus language extension features for C/C++
       # -rtlib=*             select c runtime lib with clang
@@ -5474,6 +5475,7 @@ func_mode_link ()
       -specs=*|-fsanitize=*|-fno-sanitize*|-shared-libsan|-static-libsan| \
       -ffile-prefix-map=*|-fdebug-prefix-map=*|-fmacro-prefix-map=*|-fprofile-prefix-map=*| \
       -fdiagnostics-color*|-frecord-gcc-switches| \
+      -ffast-math|-fgnu-tm|-ftree-parallelize-loops=*|-funsafe-math-optimizations|-fvtable-verify*|-shared-*| \
       -fuse-ld=*|-static-*|-fcilkplus|-Wa,*|-Werror|-Werror=*)
         func_quote_arg pretty "$arg"
 	arg=$func_quote_arg_result
